api/src/main/java/io/minio/MinioClient.java

```diff
@@ -204,40 +204,8 @@
   private static final String END_HTTP = "----------END-HTTP----------";
   private static final String US_EAST_1 = "us-east-1";
   private static final String UPLOAD_ID = "uploadId";
-
-  private static final Set<String> amzHeaders = new HashSet<>();
-
-  static {
-    amzHeaders.add("server-side-encryption");
-    amzHeaders.add("server-side-encryption-aws-kms-key-id");
-    amzHeaders.add("server-side-encryption-context");
-    amzHeaders.add("server-side-encryption-customer-algorithm");
-    amzHeaders.add("server-side-encryption-customer-key");
-    amzHeaders.add("server-side-encryption-customer-key-md5");
-    amzHeaders.add("website-redirect-location");
-    amzHeaders.add("storage-class");
-  }
-
-  private static final Set<String> standardHeaders = new HashSet<>();
-
-  static {
-    standardHeaders.add("content-type");
-    standardHeaders.add("cache-control");
-    standardHeaders.add("content-encoding");
-    standardHeaders.add("content-disposition");
-    standardHeaders.add("content-language");
-    standardHeaders.add("expires");
-    standardHeaders.add("range");
-  }
-
-  private static final Set<String> TRACE_QUERY_PARAMS = new HashSet<>();
-
-  static {
-    TRACE_QUERY_PARAMS.add("retention");
-    TRACE_QUERY_PARAMS.add("legal-hold");
-    TRACE_QUERY_PARAMS.add("tagging");
-    TRACE_QUERY_PARAMS.add(UPLOAD_ID);
-  }
+  private static final Set<String> TRACE_QUERY_PARAMS =
+      ImmutableSet.of("retention", "legal-hold", "tagging", UPLOAD_ID);
```

Comment:
I see that the variables `amzHeaders` and `standardHeaders` have been removed. Is this intentional?

Reply:
Yes. They are stale unused variables.